libsnark/zk_proof_systems/plonk/circuit.hpp

```diff
@@ -0,0 +1,69 @@
+/** @file
+*****************************************************************************
+
+Declaration of Common Preprocessed Input interfaces for ppzkSNARK
+proof system Plonk.
+
+This includes:
+- class for common preprocessed input
+
+References:
+
+\[GWC19]: "Plonk: Permutations over lagrange-bases for oecumenical
+noninteractive arguments of knowledge", Ariel Gabizon, Zachary
+J. Williamson, and Oana Ciobotaru, Cryptology ePrint Archive, Report
+2019/953, 2019, <https://eprint.iacr.org/2019/953>
+
+*****************************************************************************
+* @author     This file is part of libsnark, developed by SCIPR Lab
+*             and contributors (see AUTHORS).
+* @copyright  MIT license (see LICENSE file)
+*****************************************************************************/
+
+#ifndef PLONK_PPZKSNARK_COMMON_INPUT_HPP_
+#define PLONK_PPZKSNARK_COMMON_INPUT_HPP_
```

Comment:
Wrong include guards. Maybe the result of a copy and paste.

Reply:
Ah no you just seem to have renamed the file without changing the include guard accordingly.

Reply:
I have not finished addressing all comments yet. One of the todo-s is to fix all include guards in all files and I haven't come to that yet. Will do soon.

Reply:
Understood, thanks.